CONTRIBUTING.md

```diff
@@ -132,6 +134,8 @@ You'll need LTS or higher NodeJS in order to contribute to the _code_ in this re
 - `jest` to run all the test files on all example implementations
 - `babel` to transpile everything so it works _regardless of your version of NodeJS_.
 
+We also use `prettier` to format the files. **Prettier is _NOT_ installed when using `npm install`**, because the CI will enforce a certain version. Instead use `npx babel-node scripts/format` to run prettier. If you want to auto-format using your editor, match the version in the GitHub Workflow `verify-code-formatting.yml`.
```

Comment:
Out of curiosity, what does babel-node do?

Reply:
Run the node process through babel! [See here](https://babeljs.io/docs/en/babel-node.html).

> babel comes with a second CLI which works exactly the same as Node.js's CLI, only it will compile ES6 code before running it.

It has this caveat:

> You should not be using babel-node in production. It is unnecessarily heavy, with high memory usage due to the cache being stored in memory. You will also always experience a startup performance penalty as the entire app needs to be compiled on the fly.

But imo that's moot for a "one-time" process that's not in the hot path.